packages/features/ee/organizations/pages/components/OtherTeamsListing.tsx

```diff
@@ -12,14 +12,12 @@ export function OtherTeamsListing() {
 
   const [errorMessage, setErrorMessage] = useState("");
 
-  const { data, isLoading } = trpc.viewer.organizations.listOtherTeams.useQuery(undefined, {
+  const { data: teams, isLoading } = trpc.viewer.organizations.listOtherTeams.useQuery(undefined, {
     onError: (e) => {
       setErrorMessage(e.message);
     },
   });
 
-  const teams = useMemo(() => data?.filter((m) => m.accepted) || [], [data]);
```

Comment:
Now that you can create teams without yourself as member, we need to be able to show teams you don't belong to, which means "not accepted" teams